src/browser-frontend/reducers/pages-reducers.js

```diff
@@ -43,20 +43,21 @@ function addPage(state, { payload: { url, parentId, background } = {} }) {
 
 function removePage(state, { payload: { pageId, withoutSelectingNextLogicalPage } }) {
   return state.withMutations((mut) => {
-    const pageIndex = state.ui.pages.displayOrder.findIndex(id => id === pageId);
-    const pageCount = state.ui.pages.displayOrder.count();
     const selectedId = state.ui.pages.selectedId;
+    const idToRemove = pageId || selectedId;
```

Comment:
It seems trickier/more expensive to get the selected page from the calling code, and if you're calling remove without specifying anything, I think it's likely that you want to remove the current page.  I'm not sure what the benefit of having a single form of an action is, but would love to find out!